src/share/poudriere/jail.sh

```diff
@@ -857,11 +857,12 @@ FreeBSD: {
 }
 EOF
 
-	pkg -o REPOS_DIR="${JAILMNT}/etc/pkg" -r ${JAILMNT}/ update
-	# Omit the man/debug/kernel and tests packages, uneeded for us.
-	pkg search -qx '^FreeBSD-.*' | grep -vE -- '-man|-dbg|-kernel-|-tests' | xargs pkg -o REPOS_DIR="${JAILMNT}/etc/pkg" -r ${JAILMNT}/ install -y
+	pkg -o REPOS_DIR="${JAILMNT}/etc/pkg" -o ABI="FreeBSD:${VERSION}:${ARCH}" -r ${JAILMNT}/ update
+	# Omit the man/debug/kernel/src and tests packages, uneeded for us.
+	pkg -o REPOS_DIR="${JAILMNT}/etc/pkg" -o ABI="FreeBSD:${VERSION}:${ARCH}" -r ${JAILMNT}/ search -qCx '^FreeBSD-.*' | grep -vE -- '-man|-dbg|-kernel-|-tests|-src-' | xargs pkg -o REPOS_DIR="${JAILMNT}/etc/pkg" -r ${JAILMNT}/ install -y
```

Comment:
I would add `-w` to `grep`, just to be on the safe side

Reply:
Pretty sure that -w doesn't change a thing on this case.